### DISCUSSION TITLE
Life Safety Zone object support

### DISCUSSION THREAD

**Discussion by GeorgePriestner-Eaton** [bacnet-stack/bacnet-stack#594] [Q&A]
Does the stack support life safety zone object type (OBJECT_LIFE_SAFETY_ZONE)? Can't see an associated .c/.h files in src.

**Comment by skarg** [comment]
I added a Life Safety Zone object type this morning in PR #595.  I copied the basic Life Safety Point object and added a few more required property values and added it to the apps/server example, along with unit testing.
